rpmbuild/copr-rpmbuild.spec

```diff
@@ -60,7 +61,9 @@ Requires: %{python_pfx}-specfile >= 0.21.0
 Requires: python3-backoff >= 1.9.0
 Requires: python3-pyyaml
 
+Requires: expect
```

Comment:
A leftover dependency on `expect`

Reply:
dropped, hehe, I realized expect already was in requires